functions/src/definitions/api/interfaces.ts

```diff
@@ -25,4 +25,64 @@ interface createChecker {
   lastVotedTimestamp?: null
 }
 
-export { createVoteRequest, updateVoteRequest, createChecker }
+interface Checker {
+  name: string
+  type: "human" | "ai"
+  isActive: boolean | null
+  pendingVoteCount: number
+  last30days: last30DaysStats
+  achievements: Achievements | null
+  level: number
+  experience: number
+}
```

Comment:
Referencing `functions/src/types.ts` there seem to be missing properties from this interface

```typescript
export type Checker = {
  name: string
  type: "human" | "ai"
  isActive: boolean | null
  isOnboardingComplete: boolean | null
  singpassOpenId: string | null
  telegramId: string | null
  whatsappId?: string | null
  level: number
  experience: number
  numVoted: number
  voteWeight: number
  numCorrectVotes: number
  numVerifiedLinks: number
  preferredPlatform: string | null
  lastVotedTimestamp: Timestamp | null
  getNameMessageId: string | null
}
```

Reply:
Am intentionally returning less data from the API interface than is stored in the DB. Cos it's only for the purposes of the webapp, so some of these are unnecessary.